packages/react-cards/src/components/Card/Card.styles.ts

```diff
@@ -1,21 +1,24 @@
 import { getGlobalClassNames, HighContrastSelector } from '@uifabric/styling';
-import { Depths } from '@uifabric/fluent-theme';
```

Comment:
Can `fluent-theme` now be removed from react-cards `package.json`?

Reply:
I think so! Thanks for the catch.

Reply:
Actually, now that I tried to do it, we can't because it is used for the demo customizations.

Reply:
That raises another question.. will Cards have "Fluent by Default"? If so, it shouldn't need `fluent-theme`, even for demos.

Reply:
That's true, let me remove that as well and update the description in this PR. `Cards` should be "Fluent by Default".